config.json.sample

```diff
@@ -1,6 +1,7 @@
 {
   "zk": "zk://127.0.0.1:2181/mesos",
   "masters": ["127.0.0.1:5050"],
+  "StateTimeout": 300,
```

Comment:
Okay. I'm going to set the field as StateTimeoutSeconds in JSON, rather than using a JSON Tag.